packages/main/src/plugin/image-registry.ts

```diff
@@ -872,6 +874,18 @@ export class ImageRegistry {
       throw Error('Unable to validate provided credentials.');
     }
   }
+
+  async searchImages(registryName: string, term: string, limit: number): Promise<ImageSearchResult[]> {
+    if (registryName === 'docker.io') {
+      registryName = 'index.docker.io';
+    }
+    if (!registryName.startsWith('http')) {
+      registryName = 'https://' + registryName;
+    }
+    const resultJSON = await got.get(`${registryName}/v1/search?q=${term}&n=${limit}`);
+    const result: ImageSearchResult[] = JSON.parse(resultJSON.body).results;
+    return result;
```

Comment:
I think usually we have linter error when we do not return directly the value
```suggestion
    return JSON.parse(resultJSON.body).results;
```